components/Notifications/Notifications.js

```diff
@@ -158,6 +169,7 @@ class Notifications extends React.PureComponent {
 Notifications.propTypes = {
   alerts: PropTypes.arrayOf(PropTypes.object),
   alertDelete: PropTypes.func,
+  intl: intlShape.isRequired,
```

Comment:
hm I'm a little bit confused, where is this property actually set? In `components/Layout/Layout.js` I see Notifications being created with the defaultprops, but intl doesn't have a default one.

Reply:
Okay. So. Unfortunately I am actually not 100% sure on this because our translations are done with `react-intl` version 2 and the current version is 5. So, the documentation on `react-intl` is not really accurate for our version. But, as far as I understand, in `main.js` we wrap everything in an `<IntlProvider>` component which will provide intl as a prop.